### PR TITLE
Add crosshairs marker to Google map in the formplayer GPS question

### PR DESCRIPTION
**Pull request by sravfeyn**
@proteusvacuum 
https://dimagi-dev.atlassian.net/browse/HI-599

The marker now looks like this

Screenshot 2019-05-15 at 12 52 45 PM

### CODE REVIEW

**Comment by proteusvacuum** [comment]
Is there a reason you have the crosshair in there? Makes it look more like you are about to launch a missile at that point than putting a pin on the map. Also I'm not sure where that pin is pointing to, is it where the crosshair is centered, or the bottom of the pin?

**Comment by sravfeyn** [comment]
Farid, you can follow the [ticket](https://dimagi-dev.atlassian.net/browse/HI-599) for more discussion around UI etc. 

@orangejenny removed the extra crosshair and kept just he pointer. A small tidbit here, we could add crosshair if we want to, but it's not possible to do it without passing the image URL via a long chain of JS functions, so I thought I would use the font-awesome icons, but there is no way to do that without putting a gmaps marker first.

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js [hunk 1]
```diff
@@ -660,6 +660,10 @@ function GeoPointEntry(question, options) {
             self.map.setZoom(self.DEFAULT.anszoom);
         }
         google.maps.event.addListener(self.map, "center_changed", self.updateCenter.bind(self));
+        var marker = new google.maps.Marker({
+            map: self.map
```
[the remark this answers is not in the document]
Comment (sravfeyn) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: What comma am I missing (?)

[the remark this answers is not in the document]
Comment (orangejenny) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: after `self.map`, we have this rule on to require trailing commas: https://eslint.org/docs/rules/comma-dangle

**Comment by orangejenny** [comment]
@sravfeyn That's sounds good, I agree it's not worth dealing with crosshairs right now.

**Review by orangejenny — APPROVED** [review]

File: corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (sravfeyn) [reply corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js]: Thanks.

**Pull request merged**